content/blog/2018/06/2018-05-14-simple-pull-request-plugin.adoc

```diff
@@ -125,42 +133,136 @@ using respective paltfroms for their repositories.
 is written which will load the "Jenkinsfile.yaml" and make use of PipelineSnippetGenerator class
 to generate Declarative pipeline code.
 . Reporting of results.
-. Plugin is using Yaml from target branch right now. (Maybe this needs some discussion, example: 
+. Plugin is using Yaml from target branch right now. (Maybe this needs some discussion, example:
 what if PR contains changes in Jenkinsfile.yaml)
-. Git Push step: To push the changes of PR to the target branch. This is implemented  
+. Git Push step: To push the changes of PR to the target branch. This is implemented
 using git-plugin, PushCommand is used for this from git-plugin. credentialId,
 branch name and repository url for intracting with Github, Bitbucket, etc
 will be taken automatically from respective plugins plugin. (Yes users have to configure in that way, you can see
```

Comment:
I am not exactly sure I understand "from respective plugins plugin (Yes users have to configure in that way ...". Maybe try to write it in a declarative way? e.g. This project's plugin obtains the `credentialId` from the <name of plugin>. It obtains the branch name and repository name from <plugin name>. What do you think?